app/schema/get-search-api-data-schema.tsx

```diff
@@ -0,0 +1,11 @@
+import z from "zod";
+
+const SearchSchema = z.object({
+  results: z.array(
+    z.object({
+      year: z.optional(z.string()),
```

Comment:
```suggestion
      year: z.string().optional(),
```

Reply:
According to the docs this yields the same result. I couldn't decide on which one so I chose one lol